streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/StateStoreMetrics.java

```diff
@@ -0,0 +1,572 @@
+            storeName,
+            FLUSH,
+            FLUSH_RATE_DESCRIPTION,
+            FLUSH_TOTAL_DESCRIPTION,
+            FLUSH_AVG_LATENCY_DESCRIPTION,
+            FLUSH_MAX_LATENCY_DESCRIPTION,
+            RecordingLevel.DEBUG,
+            streamsMetrics
+        );
+    }
+
+    public static Sensor deleteSensor(final String threadId,
+                                      final String taskId,
+                                      final String storeType,
+                                      final String storeName,
+                                      final StreamsMetricsImpl streamsMetrics) {
+        return throughputAndLatencySensor(
+            threadId,
+            taskId,
+            storeType,
+            storeName,
+            DELETE,
+            DELETE_RATE_DESCRIPTION,
+            DELETE_TOTAL_DESCRIPTION,
+            DELETE_AVG_LATENCY_DESCRIPTION,
+            DELETE_MAX_LATENCY_DESCRIPTION,
+            RecordingLevel.DEBUG,
+            streamsMetrics
+        );
+    }
+
+    public static Sensor removeSensor(final String threadId,
+                                      final String taskId,
+                                      final String storeType,
+                                      final String storeName,
+                                      final StreamsMetricsImpl streamsMetrics) {
+        return throughputAndLatencySensor(
+            threadId,
+            taskId,
+            storeType,
+            storeName,
+            REMOVE,
+            REMOVE_RATE_DESCRIPTION,
+            REMOVE_TOTAL_DESCRIPTION,
+            REMOVE_AVG_LATENCY_DESCRIPTION,
+            REMOVE_MAX_LATENCY_DESCRIPTION,
+            RecordingLevel.DEBUG,
+            streamsMetrics
+        );
+    }
+
+    public static Sensor restoreSensor(final String threadId,
+                                       final String taskId,
+                                       final String storeType,
+                                       final String storeName,
+                                       final StreamsMetricsImpl streamsMetrics) {
+        return throughputAndLatencySensor(
+            threadId,
+            taskId, storeType,
+            storeName,
+            RESTORE,
+            RESTORE_RATE_DESCRIPTION,
+            RESTORE_TOTAL_DESCRIPTION,
+            RESTORE_AVG_LATENCY_DESCRIPTION,
+            RESTORE_MAX_LATENCY_DESCRIPTION,
+            RecordingLevel.DEBUG,
+            streamsMetrics
+        );
+    }
+
+    public static Sensor expiredWindowRecordDropSensor(final String threadId,
+                                                       final String taskId,
+                                                       final String storeType,
+                                                       final String storeName,
+                                                       final StreamsMetricsImpl streamsMetrics) {
+        final Sensor sensor = streamsMetrics.storeLevelSensor(
```

Comment:
I think in KIP-444 this metric would be replaced with the task-level `dropped-records` right?

Reply:
My bad. Will fix it.

Reply:
I just realized that I need PR #7566 merged before I can fix this, because `dropped-records` is task-level sensor.

Reply:
PR #7566 was merged. Rebased the PR and fixed it.

Reply:
I could be missing something, but the name still contains `expired-window`. I thought we were going with `dropped-records` across all metrics.  But this could be a misunderstanding on my part.